master/static/migrations/20231011104015_add-exp-trials-on-del-casc.tx.down.sql

```diff
@@ -0,0 +1,63 @@
+/* 
+The following tables:
+
+    * raw_steps
+    * raw_validations
+    * experiment_snapshots
+    * checkpoints_v2
+    * trials 
+    
+all contained foreign key constraints from prior migrations. 
+In order to remove `ON DELETE CASCADE` from a previously existing constraint, we remove the constraint from the table
+and add it back without the cascade during its recreation.
+*/
+
+BEGIN; 
+
+ALTER TABLE raw_steps
+DROP CONSTRAINT steps_trial_id_fkey;
+
+ALTER TABLE raw_steps
+ADD CONSTRAINT steps_trial_id_fkey FOREIGN KEY (trial_id) REFERENCES trials(id);
+
+COMMIT;
```

Comment:
migrations already run in a transaction. what's with all the sub txs?

Reply:
Ooh gotchya, didn't know that the `.tx` naming extension wraps the migration in a transaction. Thanks!